gnovm/stdlibs/std/emit_event.gno

```diff
@@ -0,0 +1,4 @@
+package std
+
+func Emit(typ string, attrs []string) { emitEvent(typ, attrs) }
```

Comment:
I believe it's a good opportunity to include Godoc here, even if it's brief.

```suggestion
func Emit(typ string, attrs []string) { emitEvent(typ, attrs) }
```

cc @thehowl